k8s/namespaces/fees-pay/fees-register-frontend/aat.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fees-register-frontend
+spec:
+  releaseName: fees-register-frontend
+  values:
+    nodejs:
+      image: hmctspublic.azurecr.io/fees-register/frontend:prod-8e30a3d-20220214154643 #{"$imagepolicy": "flux-system:fees-register-frontend"}
```

Comment:
image looks to be duplicated of fees-register-frontend.yaml? 
Please only include specific environment differences in aat.yaml etc

Example here: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/bar/bar-api/aat.yaml